public/style/main.css

```diff
@@ -128,6 +164,127 @@ nav div.section {
   overflow-y       : hidden;
   overflow-x       : hidden;
 }
+
+/* BOTTOM-UP CONTROL PANEL MODULES */
+
+.controlModules {
+  font-size: 1.2em;
+}
+.table{
+  font-size: .8em;
+}
+
+.iconContainer {
+  border-radius    : 10px;
+  background-color : yellow;
+  border-color: black;
+  border-width     : 1px;
+  color            : black;
+  margin-bottom: 0px;
+  opacity: 1;
+  padding: 4px;
+
+}
+
+.destinationIcon {
+  font-size: 1.2em;
+}
+
+.caption{
+  font-size: .6em;
+}
+
+
+.basecol_POI {
+  position         : absolute;
+  min-height: 23%;
+  max-height: 28%;
+  left             : 0%;
+  width            : 4%;
+  bottom           : 0px;
+  background-color : yellow;
+  color            : black;
+  opacity: .8;
+  overflow-y       : hidden;
+  overflow-x       : hidden;
+  border-right: 0px solid gray;
+  text-align: center;
+    
+}
+
+.basecol_filter {
+  position         : absolute;
+  min-height: 23%;
+  max-height: 38%;
+  left             : 4.2%;
+  width            : 13%;
+  bottom           : 0px;
+  background-color : yellow;
+  color            : black;
+  opacity: .8;
+  overflow-y       : hidden;
+  overflow-x       : hidden;
+   border-right: 0px solid gray;
+
+}
+
+.basecol_scenarioTable {
+  position         : absolute;
+  min-height: 23%;
+  max-height: 38%;
+  left             : 17.4%;
+  width            : 32.6%;
+  bottom           : 0px;
+  background-color : yellow;
+  color            : black;
+  opacity: .8;
+  overflow-y       : hidden;
+  overflow-x       : hidden;
+}
+
+.basecol_routeSelector{
+  position         : absolute;
+  min-height: 23%;
+  max-height: 38%;
+  left             : 50.2%;
+  width            : 2.5%;
+  bottom           : 0px;
+  font-size        : 1.2em;
+  padding: 5px;
+  background-color : yellow;
+  color            : black;
+  opacity: .8;
+  text-align: center;
+}
+
+.basecol_routeTable{
+  position         : absolute;
+  min-height: 23%;
```

Comment:
min height should perhaps be pixel count - this is an issue because you have no idea about screen size in user case, the result of which could be excessive and too little vertical